### ISSUE TITLE
CVE-2021-23820 vulnerability

### ISSUE THREAD

**Issue by maxnikitin23** [janl/node-jsonpointer#65]
Hey guys, there is ongoing security vulnerability in jsonpointer current version. jsonpointer is a dependency of @apideck/better-ajv-errors -> workbox-build-> workbox-webpack-plugin@^7.0.0/react-scripts@5.0.1 which we use as devDependencies in our apps. Wondering if you`re aware of this and planning to remediate it? Thanks
<img width="1327" alt="Screenshot 2023-12-01 at 12 10 49 PM" src="https://github.com/janl/node-jsonpointer/assets/129768239/05303e74-e88a-426b-a03f-809ba74c30bc">

**Comment by hanselninja** [comment]
I think this is a false positive in Qwiet, the CVE in question seems to be for the [json-pointer](https://www.npmjs.com/package/json-pointer) (with a -) package instead of jsonpointer (without a -).

**Comment by marcbachmann** [comment]
Yes, [CVE-2021-23820](https://nvd.nist.gov/vuln/detail/CVE-2021-23820) is not for this package.
